src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -718,17 +725,20 @@ private void RefreshAvailableUpdatesCount()
 
         private void RefreshConsolidatablePackagesCount()
         {
-            NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
+            if (Model.IsSolution)
             {
-                await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
-                _topPanel._labelConsolidate.Count = 0;
-                var loadContext = new PackageLoadContext(ActiveSources, Model.IsSolution, Model.Context);
-                var packageFeed = await CreatePackageFeedAsync(loadContext, ItemFilter.Consolidate, _uiLogger);
-                var loader = new PackageItemLoader(
-                    loadContext, packageFeed, includePrerelease: IncludePrerelease);
+                NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
```

Comment:
Consider `FileAndForget(…)`.

Reply:
Added. 

I have also created a follow-up task to address all similar occurrences. 
https://github.com/NuGet/Home/issues/8282